enterprise_subsidy/apps/api/v1/views/subsidy.py

```diff
@@ -272,3 +307,44 @@ def destroy(self, request, *args, **kwargs):
         """
         response = super().destroy(request, kwargs['uuid'])
         return response
+
+    @extend_schema(
+        tags=['subsidy'],
+        request=SubsidyUpdateRequestSerializer,
+        responses={
+            200: SubsidySerializer,
+            403: exceptions.PermissionDenied,
+            400: exceptions.ValidationError,
+            500: exceptions.APIException,
+        },
+    )
+    def update(self, request, *args, **kwargs):
+        """
+        Update a subsidy
+
+        Endpoint Location: PUT /api/v1/subsidies/{uuid}/
+        """
+        response = super().update(request, *args, **kwargs)
+        instance = self.get_object()
+        response_serializer = SubsidySerializer(instance, context=self.get_serializer_context())
```

Comment:
ah, so this brings up that thing we discussed the other day about overriding `SubsidyUpdateRequestSerializer.to_representation()` - you can have that be something like:
```python
def to_representation(self, instance):
    return SubsidySerializer(instance)
```
That should let you change this method body back to `return super().update(request, *args, **kwargs)`.

Reply:
Note: this suggestion is maybe more a stylistic thing - the drawback to it is that the serialization of the response becomes a little less obvious ("I have to look at the request serializer to understand the _response_ serialization?").

Reply:
I agree, this way is easier to read!